static/css/main.css

```diff
@@ -37,12 +37,12 @@ figure {
 }
 
 a {
-  color: #279e6aee;
+  color: rgba(39, 158, 106, .9);
```

Comment:
Note that this is not only for Chromium support, but also for Edge (latest IE – if we consider supporting this?).
[See the compatibility table](http://caniuse.com/#search=rgba)